tests/kafkatest/tests/streams/streams_standby_replica_test.py

```diff
@@ -0,0 +1,166 @@
+        updated_configs = consumer_poll_ms + "," + retries_config + "," + request_timeout + "," + max_block_ms + extra_configs
+
+        return updated_configs
+
+    def wait_for_verification(self, processor, message, file, num_lines=1, timeout_sec=20):
+        wait_until(lambda: self.verify_from_file(processor, message, file) >= num_lines,
+                   timeout_sec=timeout_sec,
+                   err_msg="Did expect to read '%s' from %s" % (message, processor.node.account))
+
+    @staticmethod
+    def verify_from_file(processor, message, file):
+        result = processor.node.account.ssh_output("grep '%s' %s | wc -l" % (message, file), allow_fail=False)
+        return int(result)
+
+
+    def setUp(self):
+        self.zk.start()
+        self.kafka.start()
+
+    def test_standby_tasks_rebalance(self):
+
+        driver_configs = "num_messages=%s,topics=%s" % (str(self.num_messages), self.streams_source_topic)
+
+        driver = StreamsRepeatingIntegerKeyProducerService(self.test_context, self.kafka, driver_configs)
+        driver.start()
+
+        configs = self.get_configs(",sourceTopic=%s,sinkTopic1=%s,sinkTopic2=%s" % (self.streams_source_topic,
+                                                                                    self.streams_sink_topic_1,
+                                                                                    self.streams_sink_topic_2))
+
+        processor_1 = StreamsStandbyTaskService(self.test_context, self.kafka, configs)
+        processor_2 = StreamsStandbyTaskService(self.test_context, self.kafka, configs)
+        processor_3 = StreamsStandbyTaskService(self.test_context, self.kafka, configs)
+
+        processor_1.start()
+
+        self.wait_for_verification(processor_1, "ACTIVE_TASKS:6 STANDBY_TASKS:0", processor_1.STDOUT_FILE)
+
+        processor_2.start()
+
+        self.wait_for_verification(processor_1, "ACTIVE_TASKS:3 STANDBY_TASKS:3", processor_1.STDOUT_FILE)
+        self.wait_for_verification(processor_2, "ACTIVE_TASKS:3 STANDBY_TASKS:3", processor_2.STDOUT_FILE)
+
+        processor_3.start()
+
+        self.wait_for_verification(processor_1, "ACTIVE_TASKS:2 STANDBY_TASKS:2", processor_1.STDOUT_FILE)
+        self.wait_for_verification(processor_2, "ACTIVE_TASKS:2 STANDBY_TASKS:2", processor_2.STDOUT_FILE)
+        self.wait_for_verification(processor_3, "ACTIVE_TASKS:2 STANDBY_TASKS:2", processor_3.STDOUT_FILE)
+
+        processor_1.stop()
+
+        self.wait_for_verification(processor_2, "ACTIVE_TASKS:3 STANDBY_TASKS:3", processor_2.STDOUT_FILE, num_lines=2)
+        self.wait_for_verification(processor_3, "ACTIVE_TASKS:3 STANDBY_TASKS:3", processor_3.STDOUT_FILE)
+
+        processor_2.stop()
+
+        self.wait_for_verification(processor_3, "ACTIVE_TASKS:6 STANDBY_TASKS:0", processor_3.STDOUT_FILE)
+
+        processor_1.start()
+
+        self.wait_for_verification(processor_1, "ACTIVE_TASKS:3 STANDBY_TASKS:3", processor_1.STDOUT_FILE)
```

Comment:
should this be in the file twice? cf L115? (note the latest change that does not delete log files anymore if a processor is stopped -- or enable cleanup for this test)